### PR TITLE
fix README.md

### PR DESCRIPTION
**Pull request by kennethlimcp**
Not sure if this is the best fix but definitely more readeable that the current one. :D

Signed-off-by: Kenneth Lim <kennethlimcp@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,22 +38,23 @@ A fantastic guide on the Ghost workflow is here: https://github.com/TryGhost/Gho
...
+* Unordered Lists
+- Unordered Lists
+> Block Quote
+*italic*
+_italic_
+**bold**
+__bold__
+~~strikethrough~~
+[link](http://www.ghost.org)
+!\[image](https://ghost.org/assets/logos-f93942864f8c9f4a0a9b0ecd6f7f055c.png)
```
Comment (novaugust) [inline README.md]: missed one ;)

File: README.md [hunk 2]
```diff
@@ -38,22 +38,23 @@ A fantastic guide on the Ghost workflow is here: https://github.com/TryGhost/Gho
 
 Ghost-Editor is very much a WYSIWYG editor but it supports a subset of markdown as content shortcuts for those of us who are mouse adverse, specifically:
 
-\# H1
-\## H2
-\## H3
+```text
+# H1
+## H2
+## H3
```
Comment (novaugust) [inline README.md]: While you're playing around in there, want to fix the H3 code?

Comment (kennethlimcp) [inline README.md]: all fixed! 😸

**Comment by disordinary** [comment]
Thanks for the PR @kennethlimcp, it's definitely much better 😄. 

Generally we ask contributors to squash their commits before submitting the PR, but I can do that from here.

**Pull request merged**